Snakefile

```diff
@@ -0,0 +1,74 @@
+rule all:
+  input:
+    "3_plot/out/doy_plot.png"
+
+"""
+Download data from ScienceBase
+"""
+rule get_sb_data:
+    params:
+        sb_item = "5e5d0bb9e4b01d50924f2b36"
+    output:
+        sb_file = "1_fetch/out/pgdl_predictions_04_N45.50-48.00_W92.00-93.00.zip"
+    script:
+        "1_fetch/sb_get.py"
+
+rule unzip_files:
+    input:
+        sb_file = "1_fetch/out/pgdl_predictions_04_N45.50-48.00_W92.00-93.00.zip"
+    params:
+        out_dir = "1_fetch/out/"
+    output:
+        "1_fetch/out/tmp/pgdl_nhdhr_{FC091A8F-FC45-46C0-91F9-18379CF0EAAE}_temperatures.csv",
```

Comment:
I'm curious if this gets treated by Snakemake as a wildcard. And if so, if there would be a way to escape that functionality. I will try to test it out myself, but I wanted to make a comment so I don't forget.